js/app/test/slider_release.spec.ts

```diff
@@ -1,48 +0,0 @@
-import { Page, Locator } from "@playwright/test";
```

Comment:
No longer need to spin up a server for this test! We now test the change/release events in the Slider.component.spec.ts file